src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -201,7 +201,7 @@ public class PackageManagerClientViewModel : NotificationObject, IPackageInstall
         #region Properties/Fields
 
         private readonly string QUARANTINED = "quarantined";
-        public PackageManagerSearchView Owner { get; set; }
+        public Window Owner { get; set; }
```

Comment:
This is a breaking change.

Reply:
Depreciated PackageManagerSearchView Owner in favor of a new ViewModelOwner.